assets/js/util.js

```diff
@@ -248,3 +248,30 @@ function synchronizeTextareaHeights() {
 /*:: import {config} from "./config.js" */
 /*:: import {persistChoices} from "./persistence.js" */
 /*:: import {iso639Codes, iso639CodesInverse} from "./localization.js" */
+
+//  As the dynamic extending of boxes is disabled in devices with width < 768, button appears when the application
+//  is run on desktop or larger screens
+
+/*  Following is the function that provides functionality of go-to-top button */
+$(function () {
```

Comment:
This should go inside the `$(document).ready(function () {` already in `utils.js`.